appsettings.json

```diff
@@ -9,6 +9,9 @@
   "AllowedHosts": "*",
   "Authorization": {
     "Issuer": "https://idsvr.example.com/oauth/v2/oauth-anonymous",
-    "Audience": "demo-client"
+    "JWKS": "https://idsvr.example.com/oauth/v2/oauth-anonymous/jwks",
+    "Kid": "1",
+    "Alg": "RS256",
+    "Audience": ["demo-client", "demo-api"]

```

Comment:
There are a couple of things about the config that you wouldn't do in a real API, so I'd like to avoid these steps:

- Configure the demo-client audience
- Configure a hard coded kid

I think you should be able to configure the API as it was before:

```csharp
services.AddAuthentication(JwtBearerDefaults.AuthenticationScheme)
  .AddJwtBearer(options =>
  {
    options.Authority = Configuration["Authorization:Issuer"];
    options.Audience = Configuration["Authorization:Audience"];
    }
  );
```

And in the configuration file use this:

```json
{
  "Authorization": {
    "Issuer": "https://idsvr.example.com/oauth/v2/oauth-anonymous",
    "Audience": "demo-api"
  }
}
```

I expect the Microsoft API stack will read an array of audiences from the JWT and audience validation will then work fine, and the API will be better than before.